### DISCUSSION TITLE
How to set alpha of the fill area in violinplot? param `alpha` seems not work.

### DISCUSSION THREAD

**Discussion by CCHLENG** [mwaskom/seaborn#3603] [Q&A]
Hi, I am drawing a violinplot by seaborn. I want to set the fill area as alpha=0.5 and edge(edge line) alpha=1.

However, I found that the param `alpha` and `color` not work.

When I use `sns.violinplot(data=[data], color=(1, 0, 0), alpha=0.5)` or `sns.violinplot(data=[data], color=(1, 0, 0, 0.5))` , I get same figure:

![image](https://github.com/mwaskom/seaborn/assets/25116183/e6d6356d-b463-4d77-9cc1-bc1aa94d9489)

Then I tried folowing code:
```python
import numpy as np
import seaborn as sns
import matplotlib.pyplot as plt

data = np.random.normal(loc=0, scale=1, size=100)

ax = sns.violinplot(data=[data], color=(1, 0, 0))

for patch in ax.collections:
    patch.set_alpha(0.5)

plt.show()
```
It works, I get this figure. But the alpha of the edge(edge line) is also set as 0.5.
![image](https://github.com/mwaskom/seaborn/assets/25116183/a303ca46-c760-4aab-b809-fc8b62e6a603)

So how can set only the fill area as alpha=0.5? Thanks!

**Comment by thuiop** [comment]
Pretty sure that matplotlib does not support different alpha for the fill and edge colors ; nothing much to do here (except reaching down to matplotlib if you really need it). Also alpha is not supposed to be a parameter of violinplot which is why it does not work, though I suppose it could be added. I am not sure of the use case though as you would not expect them to be stacked.

**Comment by CCHLENG** [comment]
Thank you! Sometimes I can need to stack violin plots in order to compare the distribution of data. Now I set the alpha of the fill manually by saving the image in svg through adobe. Fortunately, the fill and contour lines are on different layers.

**Comment by mwaskom** [comment]
You can pass an alpha value to `fillcolor`

```python
sns.violinplot(data=[data], facecolor=(1, 0, 0, .2), linewidth=3)
```
![image](https://github.com/mwaskom/seaborn/assets/315810/573286c5-e9ae-4c40-938a-287cfc7793c0)

**Reply by CCHLENG** [reply]
It works! Thank you very much!
